docs_user/modules/proc_configuring-data-plane-nodes.adoc

```diff
@@ -1,34 +1,20 @@
 [id="configuring-data-plane-nodes_{context}"]
 
-= Configuring data plane nodes
+= Configuring isolated networks on data plane nodes
 
-A complete {rhos_prev_long} ({OpenStackShort}) cluster consists of {rhocp_long} nodes and data plane nodes. The
-former use `NodeNetworkConfigurationPolicy` custom resource (CR) to configure physical
-interfaces. Since data plane nodes are not {OpenShiftShort} nodes, a different approach to
-configure their network connectivity is used.
+Data plane nodes are configured by `openstack-operator` and its CRs. The CRs define your desired network configuration for the nodes.
 
-Instead, data plane nodes are configured by `openstack-operator` and its CRs. The CRs
-define desired network configuration for the nodes.
+Your network configuration should reflect the existing network setup. You must pull the `net_config.yaml` files from each node and reuse them when you define the `OpenStackDataPlaneNodeSet` custom resources (CRs). The format of the configuration has not changed (`os-net-config` is still being used under the hood), so you can put network templates under `edpm_network_config_template` variables, either for all nodes or per node.
 
-In case of adoption, the configuration should reflect the existing network
-setup. You should be able to pull `net_config.yaml` files from each node and
-reuse it when defining `OpenstackDataplaneNodeSet`. The format of the
-configuration hasn't changed (`os-net-config` is still being used under the
-hood), so you should be able to put network templates under
-`edpm_network_config_template` variables (either common for all nodes, or
-per-node).
+To ensure that the latest network configuration is used during the data plane adoption, you should also set `edpm_network_config_update: true` in the `nodeTemplate` attribute of the `OpenStackDataPlaneNodeSet` CR.
```

Comment:
Can this instruction be a prerequisite to this procedure, or a step in the main procedure?